javascript/javascript1/week1/dogAge.js

```diff
@@ -0,0 +1,18 @@
+//2. Goodboy-Oldboy (A dog age calculator)
+
+let dogYearOfBirth = 2016;
+let dogYearFuture = 2027;
+let shouldShowResultInDogYears = true;
```

Comment:
All those could be set as `const` as you are not reassigning them later. Remember to always use `const` until it doesn't `let` you to ;)

Reply:
I have changed all variable...